### PR TITLE
remove hb_pb targeting key for deal bids with no cpm

### PR DESCRIPTION
**Pull request by protonate**
## Type of change
- [x] Feature

### CODE REVIEW

File: src/bidmanager.js [hunk 1]
```diff
@@ -129,16 +129,18 @@ exports.addBidResponse = function (adUnitCode, bid) {
 
     //if there is any key value pairs to map do here
     var keyValues = {};
-    if (bid.bidderCode && bid.cpm !== 0) {
+    if (bid.bidderCode) {
```
Comment (mkendall07) [inline src/bidmanager.js]: Should we check for dealId or comments?

Comment (mkendall07) [inline src/bidmanager.js]: DealId or CPM*

Comment (protonate) [inline src/bidmanager.js]: 👍

**Comment by protonate** [comment]
@mkendall07 notes addressed

**Comment by mkendall07** [comment]
LGTM.

**Pull request merged**